apps/aecore/test/aec_mining_tests.erl

```diff
@@ -20,6 +20,23 @@
                     145,96,71,82,28,176,23,5,128,17,245,174,170,199,54,248,167,43,185,12,108,91,
                     107,188,126,242,98,36,211,79,105,50,16,124,227,93,228,142,83,163,126,167,206>>).
 
+%% How many times to try to generate solution before giving up.
+%% aec_pow_cuckoo:generate/3 has around a 10% failure rate for
```

Comment:
how do we know about 10% rate?

Reply:
I ran the same problem 10,000 times and counted :).